src/data/injections.js

```diff
@@ -801,14 +761,106 @@ const AVAILABLE_INJECTIONS = [
   },
   {
     id: "bug1811325",
-    platform: "all",
+    platform: "desktop",
     domain: "www.bdo.com.ph",
```

Comment:
I've changed this to be desktop only as Calin mentioned both sites work fine on Android without the intervention